<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRPy V.1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FA8072;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 191, 188);
font-size:12pt;
border-radius:12px;</string>
    </property>
    <property name="placeholderText">
     <string>                                      Digite ou Cole a Url do Site</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: navy;</string>
    </property>
    <property name="text">
     <string>Gerar QrCode</string>
    </property>
   </widget>
   <widget class="QLabel" name="qr_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_label">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>440</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
letter-spacing:4px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;;
background-color: orange;</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: green;</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
